bin/oneoff/backfill_data/channel_tokens_storage_app_id

```diff
@@ -11,9 +11,10 @@
   puts "PROCESSING: slice #{slice}..."
   ActiveRecord::Base.transaction do
     channel_tokens_batch.each do |channel_token|
-      _owner_id, channel_id = storage_decrypt_channel_id(channel_token.channel)
-      channel_token.storage_app_id = channel_id
-      channel_token.save!
+      _owner_id, storage_app_id = storage_decrypt_channel_id(channel_token.channel)
+      channel_token.storage_app_id = storage_app_id
+      save_result = channel_token.save(touch: false)
+      raise "ERROR: ID #{channel_token.id}. STORAGE_APP_ID: #{storage_app_id}." unless save_result 
```

Comment:
Not seeing it in ActiveRecord documentation. But it is elsewhere, e.g., [here](https://makandracards.com/alexander-m/40457-update-a-record-without-updating-timestamps).